concepts/conditionals-switch/.meta/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
+  "blurb": "Switches allow to control the flow of a program based on the value of a variable or expression.",
```

Comment:
Grammar improvement:

Switches control the flow of a program based on the value of a variable or expression.